chainbase/src/main/java/org/tron/core/db/CommonStore.java

```diff
@@ -34,4 +33,9 @@ public BytesCapsule get(byte[] key) {
   public boolean has(byte[] key) {
     return dbSource.getData(key) != null;
   }
+
+  @Override
+  public String getDbName() {
+    return null;
```

Comment:
this is a wrong implement.

Reply:
yes, sorry. 
This is unnecessary to add:

  @Override
  public String getDbName() {
    return null;